src/utils/yaml-utils.ts

```diff
@@ -2,6 +2,26 @@ import yaml from "yaml"
 
 import { sanitizer } from "@services/utilServices/Sanitizer"
 
+const LINE_BREAK_REGEX = "[\r\n\x0B\x0C\u0085\u2028\u2029]"
+const GLOBAL_LINE_BREAK_SEARCH = new RegExp(LINE_BREAK_REGEX, "gi")
+
+export const recursiveTrimAndReplaceLineBreaks = (obj: any): any => {
+  if (typeof obj === "string") {
+    return obj.replaceAll(GLOBAL_LINE_BREAK_SEARCH, " ").trim()
+  }
+  if (Array.isArray(obj)) {
+    return obj.map((item) => recursiveTrimAndReplaceLineBreaks(item))
+  }
+  if (typeof obj === "object" && obj !== null) {
+    const newObj: any = {}
```

Comment:
possible to use some sort of recursive types and avoid the use of `any`?

Reply:
eh i re-requested a review by accident, just ping me on slack once ready

Reply:
Fixed in https://github.com/isomerpages/isomercms-backend/pull/1351/commits/1e212c836cfdea573a5c264df2e9f977a7332f2d!